snowflake/table_snowflake_schemata.go

```diff
@@ -0,0 +1,155 @@
+package snowflake
+
+import (
+	"context"
+	"database/sql"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+)
+
+//// TABLE DEFINITION
+
+// https://docs.snowflake.com/en/sql-reference/info-schema/schemata.html
+func tableSnowflakeSchemata(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "snowflake_schemata",
+		Description: "This Information Schema view displays a row for each schema in the specified (or current) database, including the INFORMATION_SCHEMA schema itself.",
+		List: &plugin.ListConfig{
+			Hydrate: listSnowflakeSchemata,
+		},
+		Columns: snowflakeColumns([]*plugin.Column{
+			{Name: "schema_id", Type: proto.ColumnType_STRING, Description: "Id of the schema."},
```

Comment:
```suggestion
			{Name: "schema_id", Type: proto.ColumnType_STRING, Description: "ID of the schema."},
```

Reply:
It's better to capitalize `ID` in descriptions, unless you copied from Snowflake docs that do not have it capitalized